wlauto/workloads/reader/uiauto/project.properties

```diff
@@ -11,4 +11,4 @@
 #proguard.config=${sdk.dir}/tools/proguard/proguard-android.txt:proguard-project.txt
 
 # Project target.
-target=android-21
```

Comment:
This should have its own commit.  Please avoid making two changes in a single commit.